### ISSUE TITLE
Crash on creating library

### ISSUE THREAD

**Issue by uli-on** [alialbaali/Noto#16]
Noto v1. 6.0 (from F-Droid) 
Android 6.0.1 (with GMS upd. to 21.18.16, cn=M sdk=23)
Webview 94.0.4606.71
Motorola Moto G4
device not rooted

This happens when pushing the "Create library" button. The error log, caught with the Scoop app, reads like this:

```
FATAL EXCEPTION: main
Process: com.noto, PID: 29643
java.util.NoSuchElementException: Collection contains no element matching the predicate.
	at r3.l.onClick(:40)
	at android.view.View.performClick(View.java:5207)
	at com.google.android.material.button.MaterialButton.performClick()
	at android.view.View$PerformClick.run(View.java:21168)
	at android.os.Handler.handleCallback(Handler.java:746)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:148)
	at android.app.ActivityThread.main(ActivityThread.java:5443)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:728)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:618)
```

**Comment by alialbaali** [comment]
This should be fixed `1.6.1`.

**Comment by uli-on** [comment]
Yes, it's fixed, thanks!
